plugins/zoom/plugin.spec.yaml

```diff
@@ -4,14 +4,15 @@ products: [insightconnect]
 name: zoom
 title: Zoom
 description: Trigger workflows on user activity while also managing your users with the Zoom plugin
-version: 1.0.0
+version: 2.0.0
 vendor: rapid7
 support: rapid7
 status: []
+supported_versions: ["Zoom API v2.10"]
 tags: [zoom, chat]
 hub_tags:
   use_cases: [alerting_and_notifications, application_management, threat_detection_and_response, user_management]
-  keywords: [zoom, chat]
```

Comment:
Was removal of the zoom keyword intentional?

Reply:
Yes - the keyword validator was flagging it as an invalid keyword :)